test/test_torch.py

```diff
@@ -934,6 +934,18 @@ def test_max_with_inf(self):
     def test_min(self):
         self._testSelection(torch.min, min)
 
+    @unittest.skipIf(not torch.cuda.is_available(), 'no CUDA')
+    def test_large_reduction_inner(self):
```

Comment:
Do you know how long this test takes to run? If it's more than a few seconds, we should mark it `slowTest`

Reply:
it is fast, some tens of millis